packages/babel-minify/index.js

```diff
@@ -0,0 +1,5 @@
+const BabelMinify = require('babel-minify-webpack-plugin');
```

Comment:
For consistency with other middleware, let's name this variable `BabelMinifyPlugin`.